sql/core/src/test/scala/org/apache/spark/sql/SQLInsertTestSuite.scala

```diff
@@ -440,14 +476,49 @@ trait SQLInsertTestSuite extends QueryTest with SQLTestUtils {
 }
 
 class FileSourceSQLInsertTestSuite extends SQLInsertTestSuite with SharedSparkSession {
+
+  import testImplicits._
+
   override def format: String = "parquet"
+
   override protected def sparkConf: SparkConf = {
     super.sparkConf.set(SQLConf.USE_V1_SOURCE_LIST, format)
   }
+
+  test("insert by name: mismatch column name") {
```

Comment:
why does the negative test not in the base test suite?

Reply:
There are some different Exception of V1 Insert between V2 Insert. Put it in base test suite will cause test failed.

Reply:
I think there are something wrong in [`TableOutputResolver`](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/TableOutputResolver.scala#L193). The `newColPath` will be empty. But it has nothing to do with this PR.